spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -331,6 +331,361 @@ You will see the year `2017` printed in the logs. The execution status of the ta
 in the database and you can retrieve information about the task execution using the shell commands
 `task execution list` and `task execution status --id <ID_OF_TASK>` or though the Data Flow UI.
 
+[[spring-cloud-skipper-integration]]
+== Spring Cloud Skipper Integration
+
+Skipper is a tool that allows you to discover Spring Boot applications and manage their lifecycle on multiple Cloud Platforms.
+You can use Skipper standalone or integrate it with Continuous Integration pipelines to help achieve Continuous Deployment
+of applications. For more details, review the link:https://docs.spring.io/spring-cloud-skipper/docs/{skipper-version}/reference/htmlsingle/#overview[reference guide]
+for a complete overview and the feature capabilities.
+
+Before we begin setting up Skipper to use with Spring Cloud Data Flow, let's review the basics by understanding foundational
+design by which the relevant infrastructure is provisioned in Kubernetes. The tailormade
+link:https://docs.spring.io/spring-cloud-skipper/docs/{skipper-version}/reference/htmlsingle/#tour-cloud-foundry[three-minute-tour for Cloud Foundry]
+walks through the fundamentals.
+
+Next up, we will review the relevant artifacts to provision Spring Cloud Skipper in Cloud Foundry.
+
+=== Download the Spring Cloud Skipper and Shell apps
+
+[source,yaml,options=nowrap,subs=attributes]
+----
+wget http://repo.spring.io/{skipper-version-type-lowercase}/org/springframework/cloud/spring-cloud-skipper-server/{skipper-version}/spring-cloud-skipper-server-{skipper-version}.jar
+wget http://repo.spring.io/{skipper-version-type-lowercase}/org/springframework/cloud/spring-cloud-skipper-shell/{skipper-version}/spring-cloud-skipper-shell-{skipper-version}.jar
+----
+
+=== Running the Skipper Server
+Similar to SCDF-server, you can either deploy the skipper-server application on Cloud Foundry or on your local machine.
+
+Let's review the sample `manifest.yml` file to deploy the skipper-server application to Cloud Foundry.
+
+[source,yaml,options=nowrap]
+----
+---
+applications:
+- name: skipper-server
+  host: skipper-server
+  memory: 1G
+  disk_quota: 1G
+  instances: 1
+  path: <PATH TO THE DOWNLOADED SKIPPER SERVER UBER-JAR>
+env:
+    SPRING_APPLICATION_NAME: skipper-server
```

Comment:
Similar to SCDF Kubernetes, we can always disable `local` platform by setting spring.cloud.skipper.server.enableLocalPlatform to false to avoid accidental deployment using local deployer.